Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
```

Comment:
Pinning is probably the better idea, always updating from latest is handy when developing and cuts down on "update base image to ..." commits, but pinned versions mean repeatable builds and always knowing where you're building from, in particular it means that the version of the base image is tracked in git which I consider to be important.